src/test/scala/Base2ConversionTest.scala

```diff
@@ -0,0 +1,44 @@
+import org.scalatest.funsuite.AnyFunSuite
```

Comment:
file name still has a 2, you can refactor > rename in the project tab on the left